tests/functional/disaster-recovery/sc_arbiter/test_add_capacity.py

```diff
@@ -22,6 +23,7 @@
 
 @tier1
 @turquoise_squad
+@bugzilla("2174745")
```

Comment:
Do we have ODF Jira tracker DFBUG for it.  I am afraid that the integration with Bugzilla decorator is not working anymore as we moved to Jira and the automation account we used for Bugzilla is not working anymore. So we need to use jira decorator instead and point to DFBUG instead.